src/action/channel.js

```diff
@@ -346,6 +347,12 @@ class ChannelAction {
     const channel = pc.find(c => c.channelPoint === channelPoint);
     if (channel) pc.splice(pc.indexOf(channel));
   }
+
+  _formatTimeTilMaturity(numBlocks) {
+    const days = Math.floor(numBlocks / (24 * 6));
+    const hours = Math.floor((numBlocks % (24 * 6)) / 6);
+    return `${days} days and ${hours} hours`;
+  }
```

Comment:
Probably better to move to helper.js for easier unit testing

Reply:
fixed